app/lib/simulate/slack-announcement.json.erb

```diff
@@ -1,19 +1,36 @@
 {
+  "token": "Sim-123456",
```

Comment:
Hrm. Should this be in the same commit as the slack simulate changes?

Reply:
You mean squash this commit with the next one where I make changes to the `slack_controller` ?

Reply:
I think so. It seems like the bug fix and slack message format should be part of the same commit.